src/lib/features/feature-search/feature.search.e2e.test.ts

```diff
@@ -59,6 +59,10 @@ const searchFeaturesWithCursor = async (
         .expect(expectedCode);
 };
 
+const getPage = async (url: string, expectedCode = 200) => {
```

Comment:
used for hateoas based navigation instead of client crafting URLs and calling "endpoints"